api/nakadi-event-bus-api.yaml

```diff
@@ -1083,6 +1133,35 @@ definitions:
         items:
           $ref: '#/definitions/Event'
 
+  Subscription:
+    description: Subscription is a high level consumption unit.
+    properties:
+      application_id:
+        type: string
+        example: 'price-service'
+        description: The id of application that will consume from the subscription.
+      event_types:
+        type: array
+        items:
+          type: string
+        description: |
+          EventTypes to subscribe to.
+          The order is not important. Subscriptions that differ only be the order of EventTypes will be considered
+          the same and will have the same id.
+          * Currently only subscription to a single EventType is supported. Subscriptions with more than one EventType
+          in event_types property will be rejected.
+      use_case:
```

Comment:
@rcillo  I think that `use_case` is actually a better name than `group_id`, because it have some meaning. `group_id` don't have it.